src/components/MailboxThread.vue

```diff
@@ -150,6 +152,18 @@ export default {
 		layoutMode() {
 			return this.$store.getters.getPreference('layout-mode', 'vertical-split')
 		},
+		horizontalListMinWidth() {
+			return this.layoutMode === 'horizontal-split' ? 60 : this.defaultListMinWidth
```

Comment:
`defaultListMinWidth` is actually a `vertical-split` value. IMO, it can be a constant number instead of "default".